datasets/new-dataset-using-llm/prompt/qa.prompt

```diff
@@ -0,0 +1,45 @@
+system:
+You are an AI assigned to turn a given text into a QnA set.
```

Comment:
You are an AI asked to convert a given text into a QnA set.

Reply:
Updated the prompt.